admin_guide/kuryr.adoc

```diff
@@ -0,0 +1,32 @@
+[[admin-guide-kuryr]]
+= Kuryr SDN administration
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+xref:../../install_config/configuring_kuryrsdn.adoc#install-config-configuring-kuryr-sdn[Kuryr]
+(or Kuryr-Kubernetes) is one of the SDN choices for {product-title}. Kuryr uses
+OpenStack networking service - Neutron to connect pods to the network. This way
```

Comment:
Kuryr uses an OpenStack networking service, Neutron, to connect pods to the network. With this method, pods can have interconnectivity with OpenStack virtual machines (VMs), which is useful for {product-title} clusters deployed on OpenStack VMs.